drivers/adc/ad7124/ad7124.h

```diff
@@ -60,11 +60,17 @@
 #define AD7124_MAX_SETUPS	8
 /* Maximum number of channels */
 #define AD7124_MAX_CHANNELS	16
-/* Device IDs
- * The Device IDs for REV-B of boards need to be updated as 0x07 for ad7124-4
- * and 0x17 for ad7124-8 */
-#define AD7124_4_ID         0x14
-#define AD7124_8_ID	        0x16
+
+/* AD7124-4 Device IDs */
+#define AD7124_4_ID      0x06
+/* Chip ID for new silicon in AD7124-4 */
+#define AD7124_4_NEW_ID  0x07
+
+/* AD7124-8 Device IDs */
+#define AD7124_8_ID1     0x14
+#define AD7124_8_ID2     0x16
```

Comment:
what is ID1 and what is ID2 and how can the user differentiate between these? again, the naming should be more suggestive and the comment as well

Reply:
The ID can be either 
<img width="790" alt="Screenshot 2024-05-15 101849" src="https://github.com/analogdevicesinc/no-OS/assets/131138604/c2dc7395-fc7c-4b2a-9c6b-02f2edf30752">

The user per say is not choosing any ID, the intention is that if he uses a board with new silicon or the old silicon , the chip id read check should not fail

Reply:
I was referring to the 1 and 2 part. if the 1 is related to the old silicon and the 2 is related to the new silicon maybe there is a way to differentiate between them, like  AD7124_8_ID_REV_0 and AD7124_8_ID_REV_A.
But I see there is also  AD7124_8_NEW_ID. All of these values are compared against the read `id_val` but it is unclear to me what they actually represent (from chip version point of view).

Reply:
@rbolboac I found this information on ezone https://ez.analog.com/data_converters/precision_adcs/w/documents/15293/ad7124-4-8-standard-silicon-b-grade-and-w-grade
I would rename the ID1 and ID2 as standard and B/W_GRADE respectively